internal/policy/policy.go

```diff
@@ -7,4 +7,5 @@ const (
 	PolicyContainer Policy = "container"
 	PolicyScratch   Policy = "scratch"
 	PolicyRoot      Policy = "root"
+	PolicyNone      Policy = "none"
```

Comment:
See comment about this being a type alias.

Reply:
fixed